github/projects.yaml

```diff
@@ -86,3 +86,5 @@
   description: Ansible role to install configuration Intrusion Detection Systems
 - project: ansible/network
   description: Ansible collection for network devices
+- project: ansible/workshops
```

Comment:
so, we have some automation in place, that is dependent on my github ID today. Can you add me as admin to ansible/workshops namespace on github?
ansible/workshops
I think, as we are growing the number of orgs, we need to setup a dedicated 'ansible-zuul' admin user for it.

Reply:
done